ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/Services/TaskService.cs

```diff
@@ -59,10 +59,12 @@ public Task DeleteAsync(Guid id, CancellationToken cancellationToken)
             return _repository.DeleteAsync(id, cancellationToken);
         }
 
-        public async Task<TaskDTO> UpdateAsync(TaskDTO taskDto, CancellationToken cancellationToken)
+        public async Task<TaskDTO> UpdateAsync(UpdateTaskDTO updateTaskDto, CancellationToken cancellationToken)
         {
             cancellationToken.ThrowIfCancellationRequested();
+            var taskDto = _mapper.Map<TaskDTO>(updateTaskDto);
             var task = _mapper.Map<Data.Models.Task>(taskDto);
+            task.ProjectName = await _projectService.GetNameAsync(task.ProjectId, cancellationToken);
```

Comment:
You fill ProjectName now, it's correct, but still you need to launch the method and see the response. It will return empty ProjectName, correct it

Reply:
See create method one more time